Patches/Red Horse Faction Last Man Contingent/RH_LMC_CE_Patch_PawnKinds_LMM.xml

```diff
@@ -0,0 +1,243 @@
+				<xpath>Defs/PawnKindDef[
+					defName="RH_LastManMercs_TankCrewman" or
+					defName="RH_LastManMercs_Grunt"
+				]</xpath>
+				<value>
+					<li Class="CombatExtended.LoadoutPropertiesExtension">
+						<primaryMagazineCount>
+							<min>6</min>
+							<max>8</max>
+						</primaryMagazineCount>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/PawnKindDef[
+					defName="RH_LastManMercs_Defector" or
+					defName="RH_LastManMercs_Rogue" or
+					defName="RH_LastManMercs_CQB" or
+					defName="RH_LastManMercs_CQB_TierII" or
+					defName="RH_LastManMercs_Assault" or
+					defName="RH_LastManMercs_Assault_TierII" or
+					defName="RH_LastManMercs_Marksman" or
+					defName="RH_LastManMercs_Marksman_TierII" or
+					defName="RH_LastManMercs_Boss" or
+					defName="RH_LastManMercs_Elite" or
+					defName="RH_LastManMercs_Trader" or
+					defName="RH_LastManMercs_Sniper"
+				]</xpath>
+				<value>
+					<li Class="CombatExtended.LoadoutPropertiesExtension">
+						<primaryMagazineCount>
+							<min>6</min>
+							<max>8</max>
+						</primaryMagazineCount>
+						<sidearms>
+							<li>
+								<generateChance>1</generateChance>
+								<magazineCount>
+									<min>2</min>
+									<max>3</max>
+								</magazineCount>	
+								<weaponTags>
+									<li>RN_M9HRT</li>
+									<li>RN_Glock17HRT</li>
+									<li>RN_TEC9</li>
+								</weaponTags>
+							</li>
+						</sidearms>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/PawnKindDef[defName="RH_LastManMercs_Gunner"]</xpath>
+				<value>
+					<li Class="CombatExtended.LoadoutPropertiesExtension">
+						<primaryMagazineCount>
+							<min>3</min>
+							<max>4</max>
+						</primaryMagazineCount>
+						<sidearms>
+							<li>
+								<generateChance>1</generateChance>
+								<magazineCount>
+									<min>2</min>
+									<max>3</max>
+								</magazineCount>	
+								<weaponTags>
+									<li>RN_M9HRT</li>
+									<li>RN_Glock17HRT</li>
+									<li>RN_TEC9</li>
+								</weaponTags>
+							</li>
+						</sidearms>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/PawnKindDef[defName="RH_LastManMercs_Grenadier"]</xpath>
+				<value>
+					<li Class="CombatExtended.LoadoutPropertiesExtension">
+						<primaryMagazineCount>
+							<min>8</min>
+							<max>10</max>
+						</primaryMagazineCount>
+						<sidearms>
+							<li>
+								<generateChance>1</generateChance>
+								<magazineCount>
+									<min>2</min>
+									<max>3</max>
+								</magazineCount>	
+								<weaponTags>
+									<li>RN_M9HRT</li>
+									<li>RN_Glock17HRT</li>
+									<li>RN_TEC9</li>
+								</weaponTags>
+							</li>
+						</sidearms>
+					</li>
+				</value>
+			</li>
+
+			<!-- ========== Tweak minimum weaponMoney for selected pawn types, so that they actually spawn with weapons ========== -->
```

Comment:
As I understand, the format of the weaponMoney tag has changed--can you confirm if the old way still works?

Reply:
Yes, the old method of specifying `weaponMoney` min and max values still works fine.